front/app/components/admin/PostManager/components/PostTable/IdeaRow.tsx

```diff
@@ -216,79 +201,72 @@ class IdeaRow extends React.PureComponent<
     );
   };
 
-  handleData = (
-    insertCellOptions: InsertConfigurationOptions<
-      CellConfiguration<IdeaCellComponentProps>
-    >
-  ) => {
-    this.setState(({ cells }) => ({
-      cells: insertConfiguration(insertCellOptions)(cells),
-    }));
+  const onUpdateIdeaPhases = (selectedPhases) => {
```

Comment:
You could add the types of this and the topics as well. I think it's `string[]`.